contributor-ladder.md

```diff
@@ -0,0 +1,222 @@
+# Contributor Ladder Template
+
+This is a light contributor ladder template for CNCF projects that requires editing before it is ready to use. Read the markdown comments, `<!-- COMMENT -->`, for additional guidance. The raw markdown uses `TODO` to identify areas that require customization.
+
+Particularly, this file provides you with a menu of options for your project.  Most projects have 3-4 defined contributor roles. Which of the roles we've outlined below you will use will be based on what makes sense for the structure and size of your project. Other roles should be deleted, or combined into an adjacent role.
+
+
+<!-- template begins here -->
+
+
+* [Contributor Ladder](##contributor-ladder-template)
+    * [Community Participant](###contributor-participant)
+    * [Contributor](###contributor)
+    * [Organization Member](##organization-member)
+    * [Reviewer](###reviewer)
+    * [Maintainer](###maintainer)
+* [Inactivity](##inactivity)
+* [Involuntary Removal](##involuntary-removal)
+* [Stepping Down/Emeritus Process](##stepping-downemeritus-process)
+* [Contact](##contact)
```

Comment:
The links are broken because they only require a single #.

```suggestion
* [Contributor Ladder](#contributor-ladder-template)
    * [Community Participant](#contributor-participant)
    * [Contributor](#contributor)
    * [Organization Member](#organization-member)
    * [Reviewer](#reviewer)
    * [Maintainer](#maintainer)
* [Inactivity](#inactivity)
* [Involuntary Removal](#involuntary-removal)
* [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
* [Contact](#contact)
```